app/config/ApplicationConfig.scala

```diff
@@ -133,12 +135,14 @@ trait ApplicationConfig {
 
   def ivNotAuthorisedUrl: String
 
-  private def createUrl(action: String) = s"${loginUrl}/${action}?origin=ma&confidenceLevel=100&completionURL=${utils.encodeQueryStringValue(callbackUrl)}&failureURL=${utils.encodeQueryStringValue(ivNotAuthorisedUrl)}"
+  private def createUrl(action: String) = s"${loginUrl}/$action?origin=ma&confidenceLevel=200&completionURL=${utils.encodeQueryStringValue(callbackUrl)}&failureURL=${utils.encodeQueryStringValue(ivNotAuthorisedUrl)}"
 
   def ivLoginUrl = createUrl(action = "registration")
 
   def ivUpliftUrl = createUrl(action = "uplift")
 
+  def ggSignInUrl: String
```

Comment:
same here

Reply:
this is just the trait for the above, what ever one is they both are... PLEASE CAN WE MAKE AppCnfig DI'd

Reply:
At some point that would be nice